.travis.yml

```diff
@@ -7,8 +7,22 @@ script:
 
 matrix:
   include:
-  - jdk: oraclejdk8
-  - jdk: oraclejdk9
+    - name: coverity
+      script:
+        - echo "coverity scan script"
+      install:
+        - echo "coverity scan install"
+      addons:
+        coverity_scan:
+          project:
+            name: "airsonic/airsonic"
+            description: "A Free and Open Source community driven media server"
+          notification_email: "airsonic@tutanota.com"
+          build_command_prepend: mvn clean
+          build_command: mvn -DskipTests=true package
+          branch_pattern: master
```

Comment:
Same as #1051 - can we do this in maven instead?

Reply:
I would be happy to, but I don't know Maven at all :/